Packages/com.llealloo.audiolink/Runtime/Scripts/AudioLink.cs

```diff
@@ -370,13 +376,23 @@ void Start()
 
             gameObject.SetActive(true); // client disables extra cameras, so set it true
             transform.position = new Vector3(0f, 10000000f, 0f); // keep this in a far away place
-
+            initialUpdateMode = audioRenderTexture.updateMode;
 
             // Disable camera on start if user didn't ask for it
             if (!audioDataToggle)
             {
                 DisableReadback();
             }
+
+#if !UDONSHARP && !CVR_CCK_EXISTS
+            Invoke(nameof(AutoCacheAudioTarget), 1);
+#endif
+        }
+
+        void OnDestroy()
+        {
+            // makes sure that playmode doesn't permanently modify the update mode
+            audioRenderTexture.updateMode = initialUpdateMode;
```

Comment:
I wonder if you can just EditorUtility.ClearDirty(audioRenderTexture) after modifying the update mode 🤔

Reply:
It does not appear to be the case.